xija/gui_fit/app.py

```diff
@@ -282,7 +282,7 @@ def __init__(self, model, main_window):
 
     def toggle_all_data(self, state):
         checked = state == QtCore.Qt.Checked
-        for i, box in enumerate(self.check_boxes):
+        for _i, box in enumerate(self.check_boxes):
```

Comment:
Could probably just cut the enumerate ...

Reply:
Yes, but then I would need to redo all the testing.